components/Tabs/src/TabBarView/MDCTabBarView.m

```diff
@@ -103,6 +104,7 @@ - (void)setItems:(NSArray<UITabBarItem *> *)items {
     itemView.accessibilityLabel = item.accessibilityLabel;
     itemView.accessibilityHint = item.accessibilityHint;
     itemView.accessibilityIdentifier = item.accessibilityIdentifier;
+    itemView.accessibilityTraits = item.accessibilityTraits;
```

Comment:
I think this should default to `.button`:

```objc
itemView.accessibilityTraits = (item.accessibilityTraits == UIAccessibilityTraitNone) : UIAccessibilityTraitButton : item.accessibilityTraits;
```

Reply:
Done.